feature.env

```diff
@@ -16,3 +16,4 @@ LOCATIONS_API_URL=http://localhost:9091/locations-api
 COMPONENT_API_URL=http://localhost:9091/components-api
 FEEDBACK_FORM_URL=http://feedback-form
 FEATURE_FLIPPER_ENABLED=true
+LOAD_DPR_ON_STARTUP=false
```

Comment:
This is so we don't have to fire up wiremocks etc for definitions on start up of the application for testing purposes